javascript/javascript1/week4/Week4HW.js

```diff
@@ -0,0 +1,93 @@
+"use strict";
+
+var arrNames = [];
+var todoArr = [];
+
+function getReplay(command) {
+  const splitedCommand = command.split(" ");
+  const userName = splitedCommand[splitedCommand.length - 1];
+  if (command === "" || typeof command != "string") {
+    return "Please say something valid!";
+  }
+  if (command == `Hello my name is ${userName}`) {
```

Comment:
What you did is correct. There is also an alternative like below 
There are string methods like 'startsWith' and 'endsWith'.
For example 
 const commandWords = inputString.split(' ');

  if (inputString.startsWith('hello my name is')) {
    name = commandWords[commandWords.length - 1];
    return `Nice to meet you ${name}`;
  }

Reply:
thank you , I didn't know about startswith and now I know how to use it. this way has been added as a note as well.